src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,23 @@
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
 // Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    <ul>
+      {todos.map(todo => (
+        <li className="TodoList__item">
+          <TodoInfo
+            key={todo.id}
+            completed={todo.completed}
+            title={todo.title}
+            user={todo.user}
```

Comment:
just put a whole todo into props
```suggestion
          <TodoInfo todo={todo} />
```